testdata/BUILD

```diff
@@ -25,6 +25,8 @@ load("//testdata:utils.bzl", "generate_deb")
 
 exports_files(["pause.tar"])
 
+exports_files(["BUILD"])
```

Comment:
why is this needed?

Reply:
Because py_image is using the BUILD file it its data attr